.github/workflows/integrate-and-deploy.yml

```diff
@@ -46,11 +49,19 @@ jobs:
       -
         name: Establish image name
         id: image
+        env:
+          TAG_LFMERGE:
         run: |
           echo ::set-output name=NAMESPACE::sillsdev/web-languageforge
           echo ::set-output name=TAG_APP::${{ inputs.image-tag-app }}
           echo ::set-output name=TAG_PROXY::${{ inputs.image-tag-proxy }}
           echo ::set-output name=TAG_NEXT_APP::${{ inputs.image-tag-next-app }}
+          echo ::set-output name=LFMERGE_NAMESPACE::ghcr.io/sillsdev/lfmerge
+          # LfMerge tag from parameter if provided, use .json file if exists, fallback to "latest" as default
+          TAG_LFMERGE=${{ inputs.image-tag-lfmerge }}
+          TAG_LFMERGE=${TAG_LFMERGE:-$(jq -r .lfmerge other-dependencies.json)}
```

Comment:
it probably is but we need to verify `jq` is present in GHA's runners

Reply:
https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md lists `jq` version 1.6, and the 18.04 runner lists `jq` version 1.5. So we're good.

Reply:
see my other comments on this topic.